docs/collect-troubleshooting.rst

```diff
@@ -21,3 +21,82 @@ This generally occurs with forms that have a lot of complex :doc:`logic <form-lo
 Sometimes this error is related to a temporary state and can be resolved by restarting the device, `closing Collect <https://support.google.com/android/answer/9079646?hl=en#zippy=%2Cclose-apps>`_ or closing other applications.
 
 If that doesn't help, consider simplifying the form. For example, could you remove some choices from your choice list or attached document? Could you split a form with a lot of logic into two separate forms filled in at different times? Could you limit the number of repeats created in a single submission?
+
+We also encourage you to share your form and device information `on the forum <https://forum.getodk.org/>`_.
+
+.. _troubleshoot-field-repeated:
+
+XPath evaluation: type mismatch - This field is repeated
+---------------------------------------------------------
+
+This error is usually followed by a list of the repeated fields expressed as :ref:`XPath paths <xpath-paths>`:
+
+.. code-block:: xml
+
+  /data/repeat[1]/description[1];/data/repeat[2]/description[1]
+
+And the hint:
+
+.. code-block:: java
+
+  You may need to use the indexed-repeat() function to specify which value you want
+
+This error points to a form design issue. If you get the error while filling out a form, please talk to the person who asked you to fill that form out.
+
+If you are the person who built the form, you will need to change the form design to fix the issue. To understand and fix this error, start by looking at the fields listed to identify which part of the field is repeated. If the fields start with ``/data``, the issue has to do with :ref:`repeats <troubleshoot-repeated-field-repeats>`. If the fields start with ``instance('<some name>')``, the issue has to do with :ref:`looking up a value in a dataset <troubleshoot-repeated-field-dataset>`.
+
+.. warning::
+
+  The web form preview will not show this error. Instead, it will default to always using the first repeated value. This is usually not the desired behavior and can be more difficult to identify. We recommend using ODK Collect to test forms with repeats.
+
+.. _troubleshoot-repeated-field-repeats:
+
+Field in repeat used as single value
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Consider this error:
+
+.. code-block:: xml
+
+  XPath evaluation: type mismatch
+  This field is repeated:
+
+  /data/repeat[1]/description[1];/data/repeat[2]/description[1]
+
+Look at the repeated fields in the error and notice ``/data/repeat[1]`` and ``/data/repeat[2]``. This means that there is a repeat with name ``repeat`` in the form. There's an attempt to access the ``description`` field inside that repeat and to use it as a single value, for example in a ``calculate``. If there are more than one ``repeat`` instances added, you will get this crash. In your XLSForm, look for a reference to ``${description}`` outside of ``repeat`` to identify the source of the issue.
```

Comment:
Don't call the repeat repeat. Maybe children or trees. Then you can use age or species instead of description.